tools/smithy-rs-sync/src/main.rs

```diff
@@ -15,26 +16,25 @@ use std::ffi::OsStr;
 use std::path::{Path, PathBuf};
 use std::process::Command;
 use std::time::Instant;
-use structopt::StructOpt;
 
-#[derive(StructOpt, Debug)]
-#[structopt(name = "smithy-rs-sync")]
+#[derive(Parser, Debug)]
+#[clap(name = "smithy-rs-sync")]
 /// A CLI tool to replay commits from smithy-rs, generate code, and commit that code to aws-rust-sdk.
-struct Opt {
+struct Args {
     /// The path to the smithy-rs repo folder.
-    #[structopt(long, parse(from_os_str))]
+    #[clap(long, parse(from_os_str))]
```

Comment:
I'm not 100% on this, but the `parse(from_os_str)` might not be needed on the PathBufs anymore.

Reply:
AFAICS they're still needed to parse non-UTF-8 paths https://github.com/clap-rs/clap/issues/3496.